dynatrace/dynatrace_test.go

```diff
@@ -36,7 +45,7 @@ func TestNewExporter(t *testing.T) {
 			t.Error("Should not return error")
 		}
 		if got.opts.URL != "https://example.com" {
-			t.Errorf("Expected %v but got %v", "https://example.com", got.opts.URL)
+			t.Errorf("Expected %#v but got %#v", "https://example.com", got.opts.URL)
```

Comment:
The `#` adds quotes, right?

Reply:
yeah it makes it print in source code representation with quotes and escapes and such so that you can just copy and paste it directly into source code